blog/_posts/2017-09-06-Issue-253.md

```diff
@@ -0,0 +1,127 @@
+---
+title: Issue 253
+layout: post
+created_at: Thu July 13 2017
+permalink: blog/2017-09-06-Issue-253
+current: blog
+author: Marie and Ines
+twitter: nk42_
+categories: student-posts
+---
+
+
+Our Project:  [Foodsaving.world](https://foodsaving.world) in the Backend   
+Repository:   <https://github.com/yunity/foodsaving-backend>   
+
+![Retro-Meeting](/img/blog/2017/retro_meeting.jpg)
```

Comment:
please add image credits to all of your images as explained [here](https://github.com/rails-girls-summer-of-code/summer-of-code/blob/gh-pages/CONTRIBUTING.md#images)

Reply:
All done. :) Thank you!